### DISCUSSION TITLE
Thoughts about support

### DISCUSSION THREAD

**Discussion by blakemcbride** [lem-project/lem#1066] [General]
Greetings,

I'm just offering some opinions from the outside.

I like the idea of lem a great deal. I have been using GNU Emacs for more than 20 years and am a big fan of Lisp. I have long desired an emacs based on Common Lisp that catered to the Lisp community. Lem is exciting!

I use GNU Emacs daily. Unfortunately, I do not get to do much with lisp. In spite of this, however, I would like to completely switch from GNU Emacs to lem.

There is, however, one big thing holding me back - support. I have questions about the use and customization of lem. I tried Discord but got no response. There is no manual, and the notes are very limited, random, and scattered. If I was able to get timely answers to questions, I might even be interested in writing a manual!

I'd really like to see lem widely adopted, but I fear without a user manual, lem will be relegated to just a few "experts" who are willing to devote significant effort to understanding the platform.

**Comment by cxxxr** [comment]
Hi Blake McBride.

I am glad you are interesting in lem.

I am sorry that the questions I wrote in the discord were not answered.
Perhaps the timing was not right.

One of lem's goals is documentation.
https://github.com/lem-project/lem/wiki/lem's-draft-vision%2C-goals%2C-non%E2%80%90goals
Questions are welcome and should be reflected in the documentation.

I would be honored if we could improve lem together.
best regards.

**Comment by blakemcbride** [comment]
Knowing that you had to be in the scripts directory for the build to work was unclear to me.  I was unable to build the system until I stumbled upon that fact.

**Comment by cxxxr** [comment]
Hmm, I don't think that's necessary.
To build, you should be able to just run make.

**Reply by cxxxr** [reply]
And I just noticed that it is not in the readme.

**Reply by cxxxr** [reply]
The readme needs to be updated.
I would not have noticed this if you had not told me.

**Comment by vindarel** [comment]
Hi there, 

> no manual

for the matter we are in the right direction :sunglasses:  After Lem 2.0, we (new users/devs) quickly put up this website: https://lem-project.github.io/lem-page/ for how to use Lem. At least, we now have a list of features and keybindings ;) There's a little M-x help, there's still the github wiki, and various people helped to translate much if not all docstrings in japanese. So, the situation has progressed, there is some infrastructure for more.

Personally, I think that for the time being, installation instructions are well placed in the README and issues.

I also use questions asked on Discord to augment the manual.

> Discord

I can see one question indeed. I believe most of us didn't know the answer nor if it was possible at all as of today (we should have added an emoji to show you that we read the question). In your situation, I would ask on a Github discussion or issue. (I view Discord as a first means to not overcharge github issues and the pressure on devs, and github as the ultimate place to ask something).

=> I'd encourage you to try again before forging your final opinion.

> switching

yay, that would be great indeed. That being said, of course the community is tiny. But already quite diverse I would say.

> write a manual

Would you like to contribute to the website? Any more prose is welcome, and help on how to develop in other languages with the LSP.

We could also generate an EPUB from the pages.

Ultimately, we'll need to publish some technical documentation, extracted from the sources, or a high-level manual for developers (there's the tiny [this](https://lem-project.github.io/lem-page/installation/extension_modes/) and [this](https://lem-project.github.io/lem-page/development/displaying/)) . Presently, lisp developers -not users- can navigate quite well Lem's code base.

> relegated to just a few "experts" who are willing to devote significant effort to understanding the platform.

Isn't Lem easy to grok for Emacs users? They'd try M-x and explore from there.
